file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237
        throw new Error(y, { cause: u });
              ^

Error: [20-May-2025 19:25:23 UTC] PHP Deprecated:  The called constructor method for WP_Widget class in to_relax_me_widget is <strong>deprecated</strong> since version 4.3.0! Use <code>__construct()</code> instead. in /wordpress/wp-includes/functions.php on line 6121
[20-May-2025 19:25:32 UTC] PHP Deprecated:  The called constructor method for WP_Widget class in to_relax_me_widget is <strong>deprecated</strong> since version 4.3.0! Use <code>__construct()</code> instead. in /wordpress/wp-includes/functions.php on line 6121
[20-May-2025 19:25:40 UTC] PHP Deprecated:  The called constructor method for WP_Widget class in to_relax_me_widget is <strong>deprecated</strong> since version 4.3.0! Use <code>__construct()</code> instead. in /wordpress/wp-includes/functions.php on line 6121
[20-May-2025 19:28:52 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>SELECT COUNT(1) as 'county' FROM INFORMATION_SCHEMA.STATISTICS
                WHERE table_schema=DATABASE() AND table_name='wp_zhu_log' AND index_name='AKZhu_log_log_date_IDX'</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT COUNT ( 1 ) as 'county' FROM _wp_sqlite_mysql_information_schema_STATISTICS WHERE `table_schema` = DATABASE ( ) AND `table_name` = 'wp_zhu_log' AND `index_name` = 'AKZhu_log_log_date_IDX' | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such function: DATABASE
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('SELECT COUNT(1)...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('SELECT COUNT(1)...')
#3 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SELECT COUNT(1)...')
#4 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(217): wpdb->get_results('SELECT COUNT(1)...')
#5 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(34): zhu_dt_log::create_database_table()
#6 /wordpress/wp-content/plugins/zhu-development-tools/lib/zhu_dt.class.php(31): zhu_dt_log::on_plugin_activation()
#7 /wordpress/wp-includes/class-wp-hook.php(324): zhu_dt::plugin_activation(false)
#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zhu-de...', false)
#11 /internal/eval.php(20): activate_plugin('zhu-development...')
#12 {main}</pre>
 for query SELECT COUNT(1) as 'county' FROM INFORMATION_SCHEMA.STATISTICS
                WHERE table_schema=DATABASE() AND table_name='wp_zhu_log' AND index_name='AKZhu_log_log_date_IDX' made by activate_plugin, do_action('activate_zhu-development-tools/zhu-dev-tools.php'), WP_Hook->do_action, WP_Hook->apply_filters, zhu_dt::plugin_activation, zhu_dt_log::on_plugin_activation, zhu_dt_log::create_database_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:28:52 UTC] PHP Notice:  Undefined offset: 0 in /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php on line 218
[20-May-2025 19:28:52 UTC] PHP Notice:  Trying to get property 'county' of non-object in /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php on line 218
[20-May-2025 19:28:52 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
MySQL query not supported. Cause: statement type: "createStatement" > "createIndex"
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(848): WP_SQLite_Driver->new_not_supported_exception('statement type:...')
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(669): WP_SQLite_Driver->execute_mysql_query(Object(WP_Parser_Node))
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('CREATE INDEX AK...')
#3 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('CREATE INDEX AK...')
#4 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(221): WP_SQLite_DB->query('CREATE INDEX AK...')
#5 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(34): zhu_dt_log::create_database_table()
#6 /wordpress/wp-content/plugins/zhu-development-tools/lib/zhu_dt.class.php(31): zhu_dt_log::on_plugin_activation()
#7 /wordpress/wp-includes/class-wp-hook.php(324): zhu_dt::plugin_activation(false)
#8 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#9 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#10 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zhu-de...', false)
#11 /internal/eval.php(20): activate_plugin('zhu-development...')
#12 {main}</pre>
 for query CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date) made by activate_plugin, do_action('activate_zhu-development-tools/zhu-dev-tools.php'), WP_Hook->do_action, WP_Hook->apply_filters, zhu_dt::plugin_activation, zhu_dt_log::on_plugin_activation, zhu_dt_log::create_database_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:29:01 UTC] PHP Notice:  Undefined property: stdClass::$county in /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php on line 218
[20-May-2025 19:29:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown create type: INDEX.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(221): WP_SQLite_DB->query('CREATE INDEX AK...')
#2 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(34): zhu_dt_log::create_database_table()
#3 /wordpress/wp-content/plugins/zhu-development-tools/lib/zhu_dt.class.php(31): zhu_dt_log::on_plugin_activation()
#4 /wordpress/wp-includes/class-wp-hook.php(324): zhu_dt::plugin_activation(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zhu-de...', false)
#8 /internal/eval.php(20): activate_plugin('zhu-development...')
#9 {main}</pre>
 for query CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date) made by activate_plugin, do_action('activate_zhu-development-tools/zhu-dev-tools.php'), WP_Hook->do_action, WP_Hook->apply_filters, zhu_dt::plugin_activation, zhu_dt_log::on_plugin_activation, zhu_dt_log::create_database_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:29:10 UTC] PHP Notice:  Undefined property: stdClass::$county in /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php on line 218
[20-May-2025 19:29:10 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date)</p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown create type: INDEX.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(221): WP_SQLite_DB->query('CREATE INDEX AK...')
#2 /wordpress/wp-content/plugins/zhu-development-tools/lib/tools/zhu_dt_log.class.php(34): zhu_dt_log::create_database_table()
#3 /wordpress/wp-content/plugins/zhu-development-tools/lib/zhu_dt.class.php(31): zhu_dt_log::on_plugin_activation()
#4 /wordpress/wp-includes/class-wp-hook.php(324): zhu_dt::plugin_activation(false)
#5 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#6 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#7 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zhu-de...', false)
#8 /internal/eval.php(20): activate_plugin('zhu-development...')
#9 {main}</pre>
 for query CREATE INDEX AKZhu_log_log_date_IDX USING BTREE ON wp_zhu_log (log_date) made by activate_plugin, do_action('activate_zhu-development-tools/zhu-dev-tools.php'), WP_Hook->do_action, WP_Hook->apply_filters, zhu_dt::plugin_activation, zhu_dt_log::on_plugin_activation, zhu_dt_log::create_database_table, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:29:19 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3_os::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-o...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-os/zia3_os...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[20-May-2025 19:29:27 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3_os::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-o...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-os/zia3_os...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[20-May-2025 19:29:34 UTC] PHP Fatal error:  Uncaught TypeError: call_user_func_array(): Argument #1 ($callback) must be a valid callback, non-static method zia3_os::flush() cannot be called statically in /wordpress/wp-includes/class-wp-hook.php:324
Stack trace:
#0 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)
#1 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#2 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_zia3-o...', false)
#3 /internal/eval.php(20): activate_plugin('zia3-os/zia3_os...')
#4 {main}
  thrown in /wordpress/wp-includes/class-wp-hook.php on line 324
[20-May-2025 19:30:41 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
        </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: SELECT `bot_id` FROM `teleadmin_bots` WHERE `TIMESTAMPDIFF` ( `SECOND` , `last_poll` , CURRENT_TIMESTAMP ) &gt; 30 AND `webhook` = 0 | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
SQLSTATE[HY000]: General error: 1 no such column: SECOND
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite-ast/class-wp-sqlite-driver.php(683): WP_SQLite_Driver->new_driver_exception('SQLSTATE[HY000]...', 'HY000', Object(PDOException))
#1 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(491): WP_SQLite_Driver->query('\n            SE...')
#2 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(432): WP_SQLite_DB->_do_query('\n            SE...')
#3 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('\n            SE...')
#4 /wordpress/wp-content/plugins/teleadmin/src/BotWebhook.php(45): wpdb->get_col('\n            SE...')
#5 /wordpress/wp-includes/class-wp-hook.php(324): TeleAdmin\BotWebhook->check_polling('')
#6 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#7 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#8 /wordpress/wp-settings.php(727): do_action('init')
#9 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#10 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#11 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#12 {main}</pre>
 for query 
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
         made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, TeleAdmin\BotWebhook->check_polling, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:30:51 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
        </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: 
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) &gt; 30
            AND webhook = 0
         | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: SECOND.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('\n            SE...')
#2 /wordpress/wp-content/plugins/teleadmin/src/BotWebhook.php(45): wpdb->get_col('\n            SE...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): TeleAdmin\BotWebhook->check_polling('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query 
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
         made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, TeleAdmin\BotWebhook->check_polling, WP_SQLite_DB->query, WP_SQLite_DB->print_error
[20-May-2025 19:31:01 UTC] WordPress database error <div style="clear:both">&nbsp;</div>
<div class="queries" style="clear:both;margin-bottom:2px;border:red dotted thin;">
<p>MySQL query:</p>
<p>
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
        </p>
<p>Queries made or created this session were:</p>
<ol>
<li>Executing: BEGIN | (no parameters)</li>
<li>Executing: 
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) &gt; 30
            AND webhook = 0
         | (no parameters)</li>
<li>Executing: ROLLBACK | (no parameters)</li>
</ol>
</div>
<div style="clear:both;margin-bottom:2px;border:red dotted thin;" class="error_message" style="border-bottom:dotted blue thin;">
Error occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such column: SECOND.
</div>
<p>Backtrace:</p>
<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()
#1 /wordpress/wp-includes/class-wpdb.php(3107): WP_SQLite_DB->query('\n            SE...')
#2 /wordpress/wp-content/plugins/teleadmin/src/BotWebhook.php(45): wpdb->get_col('\n            SE...')
#3 /wordpress/wp-includes/class-wp-hook.php(324): TeleAdmin\BotWebhook->check_polling('')
#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)
#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /wordpress/wp-settings.php(727): do_action('init')
#7 /wordpress/wp-config.php(102): require_once('/wordpress/wp-s...')
#8 /wordpress/wp-load.php(50): require_once('/wordpress/wp-c...')
#9 /internal/eval.php(3): require_once('/wordpress/wp-l...')
#10 {main}</pre>
 for query 
            SELECT bot_id
            FROM teleadmin_bots
            WHERE TIMESTAMPDIFF(SECOND, last_poll, NOW()) > 30
            AND webhook = 0
         made by require_once('wp-load.php'), require_once('wp-config.php'), require_once('wp-settings.php'), do_action('init'), WP_Hook->do_action, WP_Hook->apply_filters, TeleAdmin\BotWebhook->check_polling, WP_SQLite_DB->query, WP_SQLite_DB->print_error

    at Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237:15)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
    at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
  [cause]: Error: Error when executing the blueprint step #1 ({"step":"installPlugin","pluginData":{"resource":"wordpress.org/plugins","slug":"thumbnails-in-admin-post-listing"},"options":{"activate":true}}) : Could not download "https://downloads.wordpress.org/plugin/thumbnails-in-admin-post-listing.latest-stable.zip".
  				Check if the URL is correct and the server is reachable.
  				If it is reachable, the server might be blocking the request.
  				Check the browser console and network tabs for more information.
  
  				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
  
  				This means the server that hosts your file does not allow requests from other sites
  				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
  				cross-origin file downloads. Learn more about CORS at
  				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
  
  				If your file is on GitHub, load it from "raw.githubusercontent.com".
  				Here's how to do that:
  
  				1. Start with the original GitHub URL of the file. For example:
  				https://github.com/username/repository/blob/branch/filename.
  				2. Replace "github.com" with "raw.githubusercontent.com".
  				3. Remove the "/blob/" part of the URL.
  
  				The resulting URL should look like this:
  				https://raw.githubusercontent.com/username/repository/branch/filename
  
  				Error:
  				Error: Could not download "https://downloads.wordpress.org/plugin/thumbnails-in-admin-post-listing.latest-stable.zip"
      at Object.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12449:19)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async ms (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:12552:3)
      at async Object.onBind (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:227:48)
      at async V (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:64:10)
      at async ja (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/cli.js:8161:12) {
    [cause]: Error: Could not download "https://downloads.wordpress.org/plugin/thumbnails-in-admin-post-listing.latest-stable.zip".
    				Check if the URL is correct and the server is reachable.
    				If it is reachable, the server might be blocking the request.
    				Check the browser console and network tabs for more information.
    
    				## Does the console show the error "No 'Access-Control-Allow-Origin' header"?
    
    				This means the server that hosts your file does not allow requests from other sites
    				(cross-origin requests, or CORS).	You need to move the asset to a server that allows
    				cross-origin file downloads. Learn more about CORS at
    				https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS.
    
    				If your file is on GitHub, load it from "raw.githubusercontent.com".
    				Here's how to do that:
    
    				1. Start with the original GitHub URL of the file. For example:
    				https://github.com/username/repository/blob/branch/filename.
    				2. Replace "github.com" with "raw.githubusercontent.com".
    				3. Remove the "/blob/" part of the URL.
    
    				The resulting URL should look like this:
    				https://raw.githubusercontent.com/username/repository/branch/filename
    
    				Error:
    				Error: Could not download "https://downloads.wordpress.org/plugin/thumbnails-in-admin-post-listing.latest-stable.zip"
        at Ct.resolve (file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/blueprints/index.js:1472:13)
        at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
        at async _.run (file:///home/runner/work/playground-tester/playground-tester/node_modules/@php-wasm/util/index.js:45:14)
  }
}

Node.js v23.11.0
